woocommerce/woocommerce-android
# Activate: Local notification scheduling

Fixes #9013, which is a subtask of #8999.

This PR removes the old login notification scheduling and adds a mechanism to schedule multiple local notifications.

**To test:**

For the ease of testing, I added some temporary code, which is marked with `TODO: Remove`. It's only accessible in debug builds so there is no risk of contaminating a release.

1. Start the app
2. Turn on the flight mode
3. Tap on the offline status banner
4. Notice a message is logged saying a notification has been SCHEDULED
5. Notice that after 10 seconds, a local test notification appears
6. Notice a message is logged saying a notification has been DISPLAYED
7. Tap on the notification
8. Notice a message is logged saying a notification has been TAPPED
9. Trigger the notification again and wait 10 seconds
10. Dismiss the notification
11. Notice a message is logged saying a notification has been DISMISSED
